scripts/services/dependencies.ts

```diff
@@ -146,12 +149,16 @@ export const dependencies: T.ExpectedExports.dependencies = {
       if (!config.rpc.enable) {
         return { error: 'Must have RPC enabled' };
       }
+      if (!config.wallet.enable) {
+        return { error: 'Must have Bitcoin core wallet loaded and wallet RPC calls enabled' };
+      }
```

Comment:
Do we enable wallet rpcs in proxy autoconfig?

Reply:
> 

yes